src/Reflex/Patch.hs

```diff
@@ -5,11 +5,14 @@
 {-# LANGUAGE LambdaCase #-}
 {-# LANGUAGE StandaloneDeriving #-}
 {-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
 module Reflex.Patch
   ( Patch (..)
   , PatchDMap (..)
   , ComposeMaybe (..)
   , PatchMap (..)
+  , Group (..), Additive
```

Comment:
If you export with (..) for a class that doesn't have any members you get a warning. I can put it on another line tho.